inspec/a2-api-integration/controls/ingest.rb

```diff
@@ -1,27 +1,42 @@
+require_relative '../../constants'
+
 title 'ingest-service REST API integration tests'
 
 control 'ingest-status' do
   title 'check ingest status endpoints'
   desc 'Verifies existing status queries work similar to the A1 data-collector'
 
   describe 'GET status response' do
+    
     before(:all) do
-      test_token_request = automate_api_request(
-        '/api/v0/auth/tokens',
+      INGEST_TOKEN_ID = "ingest_test_token-#{TIMESTAMP}"
```

Comment:
these were failing because on v1 tokens all got access to `ingest:*` thanks to a default policy.
on v2, tokens must first be added to the ingest policy to get access to `ingest` APIs